3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/EtlPipeline.scala

```diff
@@ -35,11 +32,8 @@ import scalaz._
 import Scalaz._
 
 // This project
-import adapters.AdapterRegistry
-import enrichments.{
-  EnrichmentRegistry,
-  EnrichmentManager
-}
+import adapters.{AdapterRegistry,      RawEvent}
```

Comment:
Agreed. In this instance the alignment is a bit weird. There are three options: 
1. Change settings in scalafmt for all files
2. Switch off formatting locally there
3. In this particular case, move the ```import outputs.EnrichedEvent``` between the two multi imports so that scalafmt won't try to align them.

Reply:
What implications has first option?

Reply:
It will affect formatting in all files. If you want to experiment with it, just modify ```.scalafmt.conf``` on the root of SCE and run ```sbt clean compile```

Reply:
can we fix this in the end?

Reply:
What is happening with this? I thought R100 was code complete...

Reply:
this was fixed afaict

Reply:
Hey, sorry i did not respond to that comment. I thought that this is all resolved (https://github.com/snowplow/snowplow/pull/3599#issuecomment-365356824). Yes that was addressed along with all other formatting